packages/chat-core/src/index.ts

```diff
@@ -202,22 +202,9 @@ export const shouldReinitialize = (
   config1: Config,
   config2: Config,
 ): boolean => {
-  return (
-    !equals(config1.botUrl, config2.botUrl) ||
-    !equals(config1.userId, config2.userId) ||
-    !equals(config1.conversationId, config2.conversationId) ||
-    !equals(config1.languageCode, config2.languageCode) ||
-    !equals(
-      config1.experimental?.channelType,
-      config2.experimental?.channelType,
-    ) ||
-    !equals(
-      config1.experimental?.completeBotUrl,
-      config2.experimental?.completeBotUrl,
-    ) ||
-    !equals(config1.headers, config2.headers) ||
-    !equals(config1.responses, config2.responses) ||
-    !equals(config1.environment, config2.environment)
+  return !equals(
+    omit(["failureMessage"], config1),
```

Comment:
@peterszerzo actually, based on usage, I guess this should reinitialize if config changes in any way, so `!equals(config1, config2)` should be correct.

Reply:
@michaelglass you're right, let's have it do that, feel free to open a PR :)